containers/latex/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "latex",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": {
+            "VARIANT": "ubuntu-20.04"
+        }
+    },
+    "runArgs": [
+        "--cap-add=SYS_PTRACE",
+        "--security-opt",
+        "seccomp=unconfined"
+    ],
+
+    // Set *default* container specific settings.json values on container create.
+    "settings": {},
```

Comment:
Community feedback - you might add a setting to this for the latex workshop autoBuild as the “onFileChange” option fires repeatedly while Codespaces auto saves editors by default.

"latex-workshop.latex.autoBuild.run": "never"

Reply:
Thanks for the feedback! I've added this setting